appcues/src/main/java/com/appcues/trait/ContentHolderTrait.kt

```diff
@@ -54,19 +54,22 @@ public interface ContentHolderTrait : ExperienceTrait {
         /**
          * set the pagination data based on custom [CreateContentHolder] implementation
          */
-        public fun setPaginationData(paginationData: AppcuesPaginationData) {
-            appcuesPaginationData.value = paginationData
+        @Composable
```

Comment:
this is bad news, somehow we need to update the pagination data, that includes changing this method to be composable and this is a change in our API.

Reply:
Is the change in function name just to get upper case start for Composable convention?

Maybe this becomes a 3.0 release, we'll have to chat about that, i guess so. It's certainly not part of the public API that I suspect anyone is using, but still. But that's ok.

Reply:
PRing this to feature/embeds instead of main for now!